ja/calendar/chrome/lightning/lightning.dtd

```diff
@@ -37,8 +37,20 @@
 <!ENTITY lightning.menu.eventtask.accesskey		"n">
 
 <!-- properties dialog, calendar creation wizard -->
+<!-- LOCALIZATON NOTE(lightning.calendarproperties.email.label,
+     lightning.calendarproperties.forceEmailScheduling.label)
+     These strings are used in the calendar wizard and the calendar properties dialog, but are only
+     displayed when setting/using a caldav calendar -->
 <!ENTITY lightning.calendarproperties.email.label		"メール:">
 
+<!ENTITY lightning.calendarproperties.forceEmailScheduling.label	"メールクライアント側のスケジュールを優先する">
+<!-- LOCALIZATON NOTE(lightning.calendarproperties.forceEmailScheduling.tooltiptext1,
+     lightning.calendarproperties.forceEmailScheduling.tooltiptext2)
+     - tooltiptext1 is used in the calendar wizard when setting a new caldav calendar
+     - tooltiptext2 is used in the calendar properties dialog for caldav calendars -->
+<!ENTITY lightning.calendarproperties.forceEmailScheduling.tooltiptext1	"今のところ、カレンダーサーバーで予定を管理している場合、このカレンダーをプロパティダイアログでセットアップした後にしか有効にできません。">
+<!ENTITY lightning.calendarproperties.forceEmailScheduling.tooltiptext2	"このオプションは、カレンダーサーバーで予定を管理している場合のみ利用可能です。有効にすると、予定がサーバーではなく標準のメールベースのスケジュールにフォールバックされます。">
```

Comment:
フォールバックそのままなのは、一般にはちょっとなじみがないかもしれません。
退避されます。でどうでしょうか。

Reply:
この [Bug 1299610](https://bugzilla.mozilla.org/show_bug.cgi?id=1299610) は、[RFC 6638](https://www.rfc-editor.org/rfc/rfc6638.txt) "Scheduling Extensions to CalDAV" を実装するものです。
調べてみたら、forceEmailScheduling は "7.1.  Schedule Agent Parameter" に関する機能で、RFC の文書には以下のように書かれていました。
>7.1.  Schedule Agent Parameter
>
>   Parameter Name:  SCHEDULE-AGENT
>
>   Purpose:  To specify the agent expected to deliver scheduling
>      messages to the corresponding "Organizer" or "Attendee".
> ...
>      When the value "CLIENT" is specified, that
>      indicates that the client is handling scheduling messages with the
>      calendar user itself.

データの退避ではなく、メッセージの処理をクライアント側にさせますよ、というものらしいです。

Reply:
こんな感じでどうでしょうか。

This option is only available if the calendar server handles scheduling. Enabling will allow to fall back to the standard email based scheduling instead of leaving it to the server.
→ このオプションは、カレンダーサーバーで予定を管理している場合のみ利用可能です。有効にすると、予定をサーバーではなくメールクライアント側で処理できるようにします。

Reply:
👍